src/components/post/Post.jsx

```diff
@@ -14,7 +14,16 @@ import { Portal, PortalForModal, EditModal } from "../../components";
 import parse from "html-react-parser";
 import { Comment } from "./Comment";
 import { useSelector, useDispatch } from "react-redux";
-import { deletePost } from "../../features/post/postSlice";
+import {
+  deletePost,
+  likePost,
+  unlikePost,
+  addCommentToPost,
+} from "../../features/post/postSlice";
```

Comment:
you can use [Absolute imports](https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8) here